source/docs/casper/dapp-dev-guide/writing-contracts/rust-contracts.md

```diff
@@ -84,6 +93,21 @@ The `cargo casper` command can automatically set up the project structure, as sh
 
 This section covers the process of writing a smart contract in Rust, using example code from the [counter contract](https://github.com/casper-ecosystem/counter/). This simple contract allows callers to increment and retrieve an integer. Casper provides a [contract API](https://docs.rs/casper-contract/latest/casper_contract/contract_api/index.html) within the [`casper_contract`](https://docs.rs/casper-contract/latest/casper_contract/index.html) crate.
 
+:::info
+
+Important syntax elements used frequently in Rust:
+
+[Match](https://doc.rust-lang.org/rust-by-example/flow_control/match.html)<br />
```

Comment:
I would use bullet points to list syntax elements.

Reply:
Changes made in a new commit.